go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/laziercoder/mongostore
+module github.com/jshnaidman/mongostore
```

Comment:
Please revert module name. Thanks!

Reply:
Oh sorry I changed that in my repo after I made the pull request, I didn't realize it would alter this pull request. Is it possible to just cherry pick the first commit only?

Reply:
I just force pushed those two commits out of the pull request. Should be fixed.